osu.Game/OsuGame.cs

```diff
@@ -89,7 +89,7 @@ public class OsuGame : OsuGameBase, IKeyBindingHandler<GlobalAction>
 
         public readonly Bindable<OverlayActivation> OverlayActivationMode = new Bindable<OverlayActivation>();
 
-        private OsuScreenStack screenStack;
+        public OsuScreenStack screenStack;
```

Comment:
Can be `internal` since `OsuGame` and `MusicController` are in the same assembly.

Reply:
We discussed already about this, and everything in that commit is not valid, including solution, due dependencies.